config/serviceAccount.js

```diff
@@ -0,0 +1,12 @@
+export default {
+  type: 'service_account',
+  project_id: process.env.PROJECT_ID,
+  private_key_id: process.env.PRIVATE_KEY_ID,
+  private_key: process.env.PRIVATE_KEY.replace(/\\n/g, '\n'),
```

Comment:
Any reason for this?

Reply:
Yes, the firebase private_key has many line breaks which Travis ignores and treats as a text, so this throws an error on Travis during the build but when you call the replace method on the private_key with a line break, it now applies the line breaks.